starter-code/src/clue.js

```diff
@@ -1,94 +1,65 @@
 // Characters
-/*
-mrGreen
-first_name:   Jacob
-last_name:    Green
-color:        green
-description:  He has a lot of connections
-age:          45
-image:        https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg
-occupation:   Entrepreneur
+var charactersArray = [
```

Comment:
This is a bit "workaround" since we are practicing JS methods and this was good chance to create multiple objects and push them into the array (the same as in weaponsArray and roomsArray). This works too tho.